doc/memory

```diff
@@ -114,3 +111,23 @@ in the KFS docs.
 
 The length of the final block is used to determine when EOF has been reached on
 the final block.
+
+                                 Library handle
+
+    Address Length  Description
+
+    0x0000  0x02    Library ID
+    0x0002  0x02    Library address
+
+When loading a library, the kernel looks at libraries loaded by other processes
+for one with the same library ID, and copies the entry into this process if
+found. When a process is killed, the kernel looks to see if this library is in
+use by any other processes, and unloads it if not.
+
+                                 Signal handler
+
+    Address Length  Description
+
+    0x0000  0x02    Signal handler
+    0x0002  0x01    Signal number
+    0x0003  0x01    Reserved for future use
```

Comment:
Signals will have to change to be more Unix-like, and some userspace programs may have to change to compensate for this (and some syscalls too). `createSignal` and `readSignal` will have to be removed and replaced with `signal` and `kill`, which should behave like their POSIX counterparts.